### ISSUE TITLE
Go API crashes with docker-compose up

### ISSUE THREAD

**Issue by carlzoo** [el10savio/TODO-Fullstack-App-Go-Gin-Postgres-React#10]
```
osboxes@osboxes:~/TODO-Fullstack-App-Go-Gin-Postgres-React$ docker-compose up
Starting todo-fullstack-app-go-gin-postgres-react_react-app_1 ... done
Starting todo-fullstack-app-go-gin-postgres-react_postgres_1  ... done
Starting todo-fullstack-app-go-gin-postgres-react_go-api_1    ... done
Attaching to todo-fullstack-app-go-gin-postgres-react_postgres_1, todo-fullstack-app-go-gin-postgres-react_go-api_1, todo-fullstack-app-go-gin-postgres-react_react-app_1
go-api_1     | unexpected directory layout:
go-api_1     | 	import path: _/go/src/build/api
go-api_1     | 	root: /go/src
go-api_1     | 	dir: /go/src/build/api
go-api_1     | 	expand root: /go
go-api_1     | 	expand dir: /go/src/build/api
go-api_1     | 	separator: /
postgres_1   | 
postgres_1   | PostgreSQL Database directory appears to contain a database; Skipping initialization
postgres_1   | 
postgres_1   | 2020-11-19 05:07:27.049 UTC [1] LOG:  starting PostgreSQL 13.1 (Debian 13.1-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
postgres_1   | 2020-11-19 05:07:27.074 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres_1   | 2020-11-19 05:07:27.074 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres_1   | 2020-11-19 05:07:27.077 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres_1   | 2020-11-19 05:07:27.083 UTC [23] LOG:  database system was shut down at 2020-11-19 04:55:00 UTC
postgres_1   | 2020-11-19 05:07:27.094 UTC [1] LOG:  database system is ready to accept connections
todo-fullstack-app-go-gin-postgres-react_go-api_1 exited with code 1
react-app_1  | 
react-app_1  | > frontend@0.1.0 start /app
react-app_1  | > react-scripts start
react-app_1  | 
react-app_1  | Starting the development server...
react-app_1  | 
react-app_1  | Browserslist: caniuse-lite is outdated. Please run next command `npm update`
react-app_1  | Compiled successfully!
react-app_1  | 
react-app_1  | You can now view frontend in the browser.
react-app_1  | 
react-app_1  |   Local:            http://localhost:3000/
react-app_1  |   On Your Network:  http://172.22.0.2:3000/
react-app_1  | 
react-app_1  | Note that the development build is not optimized.
react-app_1  | To create a production build, use npm run build.
react-app_1  | 

```

Not sure why there's problem with directory structure?

**Comment by el10savio** [comment]
Hi @carlzoo, 

Thanks a lot for raising the issue! I will have a look at it and see if there's an issue with the `Dockerfile`. Also, are you able to docker build & run the backend go API independently (`backend/Dockerfile`), or is there a different issue popping up in there too?

**Comment by el10savio** [comment]
Fixed in #12
